app/models/mpi_data.rb

```diff
@@ -136,6 +136,11 @@ def self.for_user(user_identity)
   # @return [String] the home_phone
   delegate :home_phone, to: :profile, allow_nil: true
 
+  # The presence or absence of multiple sec_ids
+  #
+  # @return [Boolean] presence or absence of multiple sec_ids
+  delegate :multiple_sec_ids?, to: :profile, allow_nil: true
```

Comment:
Another place I probably wouldn't bother with this function, it's higher up that we care about the presence of multiple_sec_ids, not in the parsed MPI response